src/bin-utils.js

```diff
@@ -31,6 +31,37 @@ const formatAsciiString = function(e) {
 };
 
 /**
+ * Creates an object for sending to a web worker modifying properties that are TypedArrays
+ * into seperated properties for the buffer, byteOffset, and byteLength.
+ *
+ * @param {Object} message
+ *        Object of properties and values to send to the web worker
+ * @return {Object}
+ *         Modified message with TypedArray values expanded
+ * @function transferableMessage
+ */
+const transferableMessage = function(message) {
```

Comment:
Really minor, and up for debate (not sure which is actually better), but maybe it would make sense to call this function expandTypedArrays.